Getting started.md

```diff
@@ -0,0 +1,70 @@
+**Info:**
```

Comment:
While writing a project's description, always begin with the project's name. Leverage heading tags (#) to highlight them.